doc/classes/RenderingServer.xml

```diff
@@ -1931,6 +1931,13 @@
 				Sets the visibility range values for the given geometry instance. Equivalent to [member GeometryInstance3D.visibility_range_begin] and related properties.
 			</description>
 		</method>
+		<method name="instance_reset_interpolation">
+			<return type="void" />
+			<param index="0" name="instance" type="RID" />
+			<description>
+				Resets motion vectors and other interpolated values.
```

Comment:
Does this also affect physics interpolation? This should be mentioned in the description. I would also reference `[method Node.reset_physics_interpolation]` within this description (and do the same in that method's description, reference `[method RenderingServer.instance_reset_interpolation]` there).

Reply:
Lawnjelly is currently rewriting the physics stuff and plans to remove ´reset_physics_interpolation´ entirely. Right now the main focus of this PR is to get this working only on the rendering side, physics will be added by lawnjelly in a followup PR.